app/controllers/mixins/ems_common_angular.rb

```diff
@@ -149,6 +149,8 @@ def get_task_args(ems)
         [user, params[:default_password], endpoint_opts]
       when 'ManageIQ::Providers::Lenovo::PhysicalInfraManager'
         [user, password, params[:default_hostname], params[:default_api_port], "token", false]
+      when 'ManageIQ::Providers::Hawkular::MiddlewareManager'
+        [params[:default_hostname], params[:default_api_port], user, password, params[:default_security_protocol], params[:default_tls_ca_certs]]
```

Comment:
please add `true` at the and depend on https://github.com/ManageIQ/manageiq-providers-hawkular/pull/100

Reply:
Ok I understand now, ^^ I did you a review about that in the PR